docs/03_server/04_event-handler/01_introduction.md

```diff
@@ -48,3 +49,13 @@ We recommend using **Kotlin** to implement Event Handlers.
 :::
 
 All events created in the Event Handler are exposed as HTTP/HTTPs [REST endpoints](../../../server/integration/rest-endpoints/introduction/) automatically by the Genesis platform. You can use any http client, such as postman, to access these custom endpoints.
+
+## Open API
+
+:::important
+
+Open API support was introduced in version 7.0 of the Genesis platform.
+
+:::
+
+To help with external integration and testing, [Open API](../../integration/open-api/) specifications are generated for each event handler resource.
```

Comment:
Is it possible to customize? So the link can be included here as well?

Reply:
no

Reply:
@kievitsp it will use whatever overrides we have in our json schema right? So technically we could add annotations to customise metadata at the json schema level and it will propagate to the OpenAPI level?

Reply:
@kievitsp yes, I think that would be helpful. Thanks!